app/scripts/templates/settings/communication_preferences.mustache

```diff
@@ -35,7 +35,7 @@
       <div class="button-row">
         <button type="submit" id="marketing-email-optin" class="marketing-email-optin settings-button primary" autofocus>
           {{#isOptedIn}}
-            {{#t}}Unsubscribe{{/t}}
+            {{#t}}Manage Subscriptions{{/t}}
```

Comment:
We should probably remove the {{#preferencesUrl}} link section on lines 30-32.

Here's what happens if `preferencesUrl` is defined:

<img width="504" alt="screen shot 2017-10-05 at 16 08 45" src="https://user-images.githubusercontent.com/848085/31234787-4dc07dc0-a9df-11e7-8121-317d7457afd5.png">

@ryanfeeley - thoughts?

Reply:
It seems like we should be able to [remove the JS to open the page](https://github.com/mozilla/fxa-content-server/pull/5551/files#diff-5793cc2a765a7665d71904a30e7b860bR121) and use an anchor element with `target="_blank" href="{{preferencesUrl}}"`.